maxim/tests/test_logger.py

```
@@ -27,10 +27,12 @@
 # Set global log level to debug
 logging.getLogger().setLevel(logging.DEBUG)
 
+baseUrl = os.getenv("MAXIM_BASE_URL") or "https://app.getmaxim.ai"
+
 
 class TestLoggerInitialization(unittest.TestCase):
     def setUp(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)